packages/@jsii/python-runtime/src/jsii/_kernel/providers/process.py

```diff
@@ -298,10 +298,7 @@ def stop(self):
         try:
             self._process.wait(timeout=5)
         except subprocess.TimeoutExpired:
-            try:
-                self._process.terminate(timeout=5)
-            except subprocess.TimeoutExpired:
-                self._process.kill()
+            self._process.terminate()
```

Comment:
Note: `terminate` does the same work as `kill`, except on POSIX platforms, where the signal sent to the subprocess is different (`SIGTERM` for `terminate`, `SIGKILL` for `kill`). The default handling for both of these signals is the same (terminates the process).

`SIGKILL` cannot be trapped (always kills the process immediately, not giving it a chance to clean up after itself).

`SIGTERM` can be trapped (and rescued to not terminate the process), and gives the process a modest slice of time to clean up after itself. Since we are not trapping `SIGTERM`, both signals achieve the same - terminating the `node` VM, except `SIGTERM` would allow VM exit handlers to run, which is desirable.